cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/README.md

```diff
@@ -1,2 +1,2 @@
 
-We use a datastore with datapath as input folder in this example. In first step, we will mount folder `azureml://datastores/workspaceblobstore/paths/azureml/` using `uri_file` type, then create `hello-world.txt` in it, in the second step, we will print the content in `hello-world.txt` using `uri_file` type.
\ No newline at end of file
+We use a datastore with datapath as input folder in this example. In first step, we will mount folder `azureml://datastores/workspaceblobstore/paths/azureml/` using `uri_folder` type, then create `hello-world.txt` in it, in the second step, we will print the content in `hello-world.txt` using `uri_file` type.
```

Comment:
update the folder here

Reply:
seems this folder is input path in sample_input_data which is azureml://datastores/workspaceblobstore/paths/azureml/

Reply:
I think this folder is input data sample_input_data of the first step, I will update ```then create `hello-world.txt` in it```